steward/src/cellars/adaptors/aave_v3.rs

```diff
@@ -250,9 +250,11 @@ fn get_encoded_adaptor_calls(
             MorphoAaveV3DebtTokenV1Calls(params) => {
                 calls.extend(adaptors::morpho::morpho_aave_v3_debt_token_adaptor_v1_calls(params)?)
             }
+            /*
```

Comment:
Should include a TODO comment here as well indicating why this is commented out.